openid-4-verifiable-presentations-1_0.md

```diff
@@ -460,6 +484,64 @@ To use `client_id_scheme` values `entity_id`, `did`, `verifier_attestation`, `x5
 
 Other specifications can define further values for the `client_id_scheme` parameter. It is RECOMMENDED to use collision-resistant names for such values.
 
+## Request URI Method POST {#request_uri_method_post}
+
+This request is handled by the Request URI endpoint of the Verifier.  
+
+The request MUST use the HTTP POST method with the https scheme, and the content type "application/x-www-form-urlencoded" and the accept header set to "application/oauth-authz-req+jwt". 
+
+The following parameters are defined: 
+
+`wallet_metadata`:
+: OPTIONAL. OPTIONAL. A String containing a JSON Object containing metadata parameters as defined in (#as_metadata_parameters). 
+
+`wallet_nonce`:
+: OPTIONAL. A String containing a value the Verifier MUST use in the `wallet_nonce` authorization parameter when creating the signed presentation request object. For example, a base64url encoded fresh, cryptographically random number with sufficient entropy.  
+
+If the Wallet wants the Verifier to encrypt the request object, it SHOULD use the `jwks` or `jwks_uri` parameter within `wallet_metadata` to pass the public key for the input to the key agreement. Other mechanisms to pass the encryption key can be used as well. 
```

Comment:
I'm fine with this direction but I would feel better if we could include the following `wallet_metadata` parameters as well to indicate support for encryption.
- authorization_encryption_alg_values_supported
- authorization_encryption_enc_values_supported

And for signatures:
- request_object_signing_alg_values_supported

This is also what we do in ISO 18013-7 where JARM encrption is mandatory. In ISO 18013-7, the wallet metadata is assumed to be statically pre-populated to the verifier.

Reply:
please propose text

Reply:
See proposal below.